google/cloud/spanner_v1/database.py

```diff
@@ -60,7 +60,7 @@
 from google.cloud.spanner_v1.table import Table
 
 # pylint: enable=ungrouped-imports
-
+from google.cloud.spanner_v1.types import RequestOptions
```

Comment:
You can import `RequestOptions` from `google.cloud.spanner_v1` so please move this to be with the other `google.cloud.spanner_v1` imports.